assets/wizards/dashboard/views/DevInfo/style.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../../shared/scss/colors';
+@import '~@wordpress/base-styles/colors';
+
+.newspack-dashboard__news {
+	details {
+		margin-bottom: 5px;
+		border: 1px solid $light-gray-200;
+		padding: 20px;
+		border-radius: 3px;
+		> div {
+			border-radius: 2px;
+			margin: 20px 0 0;
+			padding-left: 40px;
+			h3:first-child {
+				margin-top: 0;
+			}
+		}
+		ul {
+			list-style: disc;
+			padding-left: 18px;
+		}
+		li {
+			margin-bottom: 0;
+		}
+		ul:last-child {
+			margin-bottom: 0;
+		}
+		h3 {
+			margin: 0;
+		}
+		svg {
+			transition: transform 200ms;
+		}
+		&[open] {
+			svg {
+				transform: rotate( 180deg );
+			}
+		}
+	}
+	summary {
+		display: flex;
+		font-size: 16px;
+		cursor: pointer;
+		&:focus {
+			outline: none;
+		}
+		&::-webkit-details-marker {
+			display: none;
+		}
+		> div:first-child {
```

Comment:
This is super nitpicky, but I noticed when the summary was only one line long, the spacing on the bottom is taller than the top: 

![image](https://user-images.githubusercontent.com/177561/87738652-b7bd2380-c792-11ea-8e9c-67476764b41c.png)

Adding this should even them out: 

```suggestion
		> div:first-child {
			line-height: 1;
```

Reply:
Added in e2cd04c!